doc: samples: Change from firmware nRF Cloud to AWS S3 Bucket (#1412)
NordicPlayground/fw-nrfconnect-nrf: samples/nrf9160/http_application_update/README.rst

=========================

Before building the sample, you must specify where the image file will be located.
If you do not want to host it yourself, you can upload it to a public S3 bucket on Amazon Web Services (AWS).
See `Setting up an AWS S3 bucket`_ for instructions.

To specify the location in |SES|:

1. Select **Project** > **Configure nRF Connect SDK Project**.
#. Navigate to **HTTP application update sample** and specify the **Download host name** (``CONFIG_DOWNLOAD_HOST``) and **The file to download** (``CONFIG_DOWNLOAD_FILE``).
#. Click **Configure** to save the configuration.

.. include:: /includes/aws_s3_bucket.txt

Hosting your image on an AWS S3 Server
--------------------------------------

1. Go to `AWS S3 console`_ and sign in.

>    Otherwise, create an account and sign in.

Reviewer: Yes - they should have an account since they created the bucket.
Good catch!

>    Otherwise, create an account and sign in.

Author: Okei :)

=========================

Before building the sample, you must specify where the image file will be located.

> If you do not want to host it yourself, you can upload it to a public S3 bucket on AWS.

Reviewer: ```suggestion
If you do not want to host it yourself, you can upload it to a public S3 bucket on Amazon Web Services (AWS).
```

> If you do not want to host it yourself, you can upload it to an public S3 bucket on AWS.

Reviewer 2: "a public S3..."

Author: ```suggestion
If you do not want to host it yourself, you can upload it to a public S3 bucket on AWS.
```

> See `AWS S3 bucket setup`_ for instructions.

Reviewer: Change to new heading.

Author: Added

To specify the location in |SES|:

1. Select **Project** > **Configure nRF Connect SDK Project**.
#. Navigate to **HTTP application update sample** and specify the **Download host name** (``CONFIG_DOWNLOAD_HOST``) and **The file to download** (``CONFIG_DOWNLOAD_FILE``).
#. Click **Configure** to save the configuration.

.. include:: /includes/aws_s3_bucket.txt

Hosting your image on an AWS S3 Server
--------------------------------------

1. Go to `AWS S3 console`_ and sign in.

>    If you already have an AWS account, you can use the same credentials.

Reviewer: delete this line

Author: Both from L65-L66?

Author: Haven't added this change yet since I was unsure what to remove.

Reviewer: Only "If you already have an AWS account, you can use the same credentials."
Because of course you can. ;) This was pointed out in the context of nrfcloud vs. firmware.nrfcloud.com.

Author: okei :)

   Otherwise, create an account and sign in.
#. Go to the bucket you have created.
#. Click **Upload** and select the file ``app_update.bin`` (located in the ``zephyr`` subfolder of your build directory).

> #. Click the file you uploaded in the bucket and find the download URL to the file through the *Object URL* field.

Reviewer: ```suggestion
#. Click the file you uploaded in the bucket and check the **Object URL** field to find the download URL for the file.
```

Author: Added

> When specifying the image file, use the ``<bucket-name>.s3.<region>.amazonaws.com`` part of the URL for the download host name.

Author: ```suggestion
When specifying the image file, use the ``<bucket-name>.s3.<region>.amazonaws.com`` part of the URL for the download hostname.
```
